emailTemplates/morning-unclaimed.html

```diff
@@ -808,7 +808,7 @@ <h3 style="text-align:center;">THANK YOU!</h3>
                                                 <tr>
                                                   <td valign="top" class="mcnTextContent" style="padding-top:0; padding-right:18px; padding-bottom:9px; padding-left:18px;">
                                                     <em>Copyright © 2020 CV19 Assist, All rights reserved.</em><br>
-                                                    <u>&nbsp;<a href="*|UNSUB|*">unsubscribe from this list</a>.</u>
+                                                    <u>&nbsp;<a href="https://{{projectDomain}}/emailUnsubscribe?type=morning-unclaimed&email={{to}}">unsubscribe from this list</a>.</u>
```

Comment:
We will still need to test this to see if the email come through to this query param - I'm not sure of what the variable for the email it is going to within the email template context (since we are sending by uid)